### PR TITLE
Change CSS Modifiers knobs to multi-selects

### PR DESCRIPTION
**Pull request by jahvi**
# Related issue
#342

# Scope of work
Change CSS Modifiers knobs to use multi-selects

# Screenshots of visual changes
N/A

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/component-rules.html) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by przspa** [comment]
hi @jahvi can you checked again? Search in *.stories.js customClass: { you should find more cases, for example SfCircleIcon :)

**Comment by jahvi** [comment]
My bad I just  did a search for  "css modifiers", I'll update the PR today.

**Comment by filrak** [comment]
Good to see you again bud ;) Thank you very much!

**Comment by jahvi** [comment]
Thanks @filrak!

@przspa You were right I missed loads, I updated the PR I think I covered all of them now.

File: packages/vue/src/components/atoms/SfBadge/SfBadge.stories.js [hunk 1]
```diff
@@ -34,16 +34,16 @@ storiesOf("Atoms|Badge", module)
...
-              "sf-badge--alert",
-              "sf-badge--warning",
-              "sf-button--full-width",
-              "sf-button--full-width  sf-badge--warning"
-            ],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/atoms/SfBadge/SfBadge.stories.js]:
```suggestion
```
Can you remove all `null` from `multi-select` knobs

File: packages/vue/src/components/atoms/SfButton/SfButton.stories.js [hunk 2]
```diff
@@ -48,20 +48,20 @@ storiesOf("Atoms|Button", module)
...
-              "color-info",
-              "color-success",
-              "sf-button--full-width",
-              "sf-button--full-width"
-            ],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/atoms/SfButton/SfButton.stories.js]:
```suggestion
```

File: packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.stories.js [hunk 3]
```diff
@@ -54,26 +58,27 @@ storiesOf("Atoms|CircleIcon", module)
...
-              "color-warning",
-              "color-danger",
-              "color-info",
-              "color-success"
-            ],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.stories.js]:
```suggestion
```

File: packages/vue/src/components/atoms/SfIcon/SfIcon.stories.js [hunk 4]
```diff
@@ -74,10 +78,15 @@ storiesOf("Atoms|Icon", module)
...
       props: {
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-icon--color-primary", "sf-icon--color-secondary"],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/atoms/SfIcon/SfIcon.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfBanner/SfBanner.stories.js [hunk 5]
```diff
@@ -113,19 +117,20 @@ storiesOf("Molecules|Banner", module)
...
-              "sf-banner--center",
-              "sf-banner--secondary",
-              "sf-banner--container-medium",
-              "sf-banner--container-full"
-            ],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfBanner/SfBanner.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfCallToAction/SfCallToAction.stories.js [hunk 6]
```diff
@@ -22,14 +26,15 @@ storiesOf("Molecules|CallToAction", module)
...
-            [
-              "null",
-              "sf-call-to-action--secondary",
-              "sf-call-to-action--light"
-            ],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfCallToAction/SfCallToAction.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfCounter/SfCounter.stories.js [hunk 7]
```diff
@@ -86,11 +87,14 @@ storiesOf("Molecules|Counter", module)
...
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-counter--large"],
-            "null",
-            "CSS-Modifiers"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfCounter/SfCounter.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.stories.js [hunk 8]
```diff
@@ -63,17 +68,19 @@ storiesOf("Molecules|SearchBar", module)
...
-              "sf-search-bar--position-right",
-              "sf-search-bar--position-right-mobile",
-              "sf-search-bar--position-right-desktop"
-            ],
-            "null",
-            "CSS-Modifiers"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfSection/SfSection.stories.js [hunk 9]
```diff
@@ -43,11 +43,14 @@ storiesOf("Molecules|Section", module)
           default: number("(prop) levelHeading", 2)
         },
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-section--no-underline"],
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfSection/SfSection.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js [hunk 10]
```diff
@@ -38,10 +38,15 @@ storiesOf("Molecules|Select", module)
...
       props: {
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-select--bordered", "sf-select--underlined"],
-            "sf-select--underlined"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js [hunk 11]
```diff
@@ -97,10 +102,15 @@ storiesOf("Molecules|Select", module)
...
       props: {
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-select--bordered", "sf-select--underlined"],
-            "null"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js]:
```suggestion
```

File: packages/vue/src/components/molecules/SfSlidingSection/SfSlidingSection.stories.js [hunk 12]
```diff
@@ -35,11 +39,14 @@ storiesOf("Molecules|SlidingSection", module)
...
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-card--modifier"],
-            "null",
-            "CSS-Modifiers"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfSlidingSection/SfSlidingSection.stories.js]:
```suggestion
```

File: packages/vue/src/components/organisms/SfBannerGrid/SfBannerGrid.stories.js [hunk 13]
```diff
@@ -27,11 +31,14 @@ storiesOf("Organisms|BannerGrid", module)
...
         customClass: {
-          default: select(
-            "CSS Modifier",
-            ["null", "sf-banner-grud--modifier"],
-            "null",
-            "CSS-Modifiers"
+          default: options(
+            "CSS Modifiers",
+            {
+              null: "null",
```
Comment (przspa) [inline packages/vue/src/components/organisms/SfBannerGrid/SfBannerGrid.stories.js]:
```suggestion
```

**Comment by jahvi** [comment]
All `null` options gone

**Comment by przspa** [comment]
thanks @jahvi! great job!

**Pull request merged**